charts/icm-test/scripts/configure-azure-secret.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -eu pipefail
+
+# Create a new storage account and copy necessary files if it doesn't exist yet
+if [[ "$HAS_AZURE_STORAGE_ACCOUNT" == 'n' ]]; then
+    AZURE_STORAGE_ACCOUNT="$(uuidgen | tr -d '-' | tr 'A-Z' 'a-z' | cut -c1-24)"
+    AZURE_RESOURCE_GROUP="ateam-common"
+
+    echo "Creating new storage account with name $AZURE_STORAGE_ACCOUNT in resource group $AZURE_RESOURCE_GROUP..."
+
+    az storage account create \
```

Comment:
@DMuench-intershop : Please add some information here that this is a test storage account which needs to be deleted afterwards. Also check to maybe add tags here.